src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1256,6 +1256,15 @@ High quality may be unlocked on some videos that require high device dimensions,
 A different codec will be applied after about 20 seconds of buffering."</string>
     <string name="revanced_reject_av1_codec_toast">Fallback process causes about 20 seconds of buffering.</string>
     <string name="revanced_remember_playback_speed_toast">Changing default speed to %s.</string>
+    <string name="revanced_remember_video_quality_last_selected_summary_off">Quality changes only apply to the current video.</string>
+    <string name="revanced_remember_video_quality_last_selected_summary_on">Quality changes apply to all videos.</string>
+    <string name="revanced_remember_video_quality_last_selected_title">Remember video quality changes</string>
+    <string name="revanced_show_toast_remember_video_quality_title">Show a toast when changing the default video quality</string>
```

Comment:
The string in the Preference title is too long

Most devices cannot display this in one line, so it will be displayed as `...`